documentation/staging/content/samples/simple/azure-kubernetes-service/domain-on-pv.md

```diff
@@ -105,7 +105,7 @@ secret/domain1-weblogic-credentials labeled
 The secret domain1-weblogic-credentials has been successfully created in the default namespace.
 ```
 
-We will use the `kubernetes/samples/scripts/create-kuberetes-secrets/create-docker-credentials-secret.sh` script to create the Docker credentials as a Kubernetes secret. Please run:
+We will use the `kubernetes/samples/scripts/create-kubernetes-secrets/create-docker-credentials-secret.sh` script to create the Docker credentials as a Kubernetes secret. Please run:
```

Comment:
do we need to rename that directory too? (perhaps it is already renamed as part of this PR but I couldn't tell for sure).

Reply:
It does look like we need to use `git mv` to fix the name of the directory too.

Reply:
Good catch! Sorry - I forgot.